### PR TITLE
stdout/stderr log filtering for windows

### PR DESCRIPTION
**Pull request by sarahpeiffer**
This PR adds the FILTER plugin for windows fluent bit container logs. 

Validation/testing:
(tested on cluster with docker and containerd windows nodes. logging yaml applied outputting logs per second w/ 2k log size, 1,800 logs per minute)
- Confirmed setting log_collection_settings.stdout enabled to false stops collection of stdout logs (but not stderr)
- Confirmed setting log_collection_settings.stderr enabled to false stops collection of stderr logs (but not stdout)
- Confirmed setting both stdout and stderr to false with no namespaces in the excluded list stops all logs
- Confirmed setting log_collection_settings.stdout enabled to true and filtering out a stdout namespace works as expected
- Switched between setting stdoutenabled to true and stdoutenabled to false multiple times with expected behavior
- Checked that changing the omsagent-win image does not duplicate logs since fluent bit state is maintained (this was expected just confirming before release)
- Checked that setting log_collection_settings.enrich_container_logs to true shows container image and container name as expected

This PR also includes some changes to fix build breaks for PRs that were already in ci_dev

### CODE REVIEW

**Comment by ganga1980** [comment]
> This PR adds the FILTER plugin for windows fluent bit container logs.
> 
> Validation/testing: (tested on cluster with docker and containerd windows nodes. logging yaml applied outputting logs per second w/ 2k log size, 1,800 logs per minute)
> 
> * Confirmed setting log_collection_settings.stdout enabled to false stops collection of stdout logs
> * Confirmed setting log_collection_settings.stderr enabled to false does not stop collection of stdout logs
> * Confirmed setting log_collection_settings.stdout enabled to true and filtering out a stdout namespace works as expected
> * Switched between setting stdoutenabled to true and stdoutenabled to false multiple times with expected behavior
> * Checked that changing the omsagent-win image does not duplicate logs since fluent bit state is maintained (this was expected just confirming before release)
> * Checked that setting log_collection_settings.enrich_container_logs to true shows container image and container name as expected

> Thanks for thorough validation and explanation what's being tested.

File: build/common/installer/scripts/tomlparser.rb [hunk 1]
```diff
@@ -260,6 +260,16 @@ def get_command_windows(env_variable_name, env_variable_value)
   file = File.open("setenv.ps1", "w")
 
   if !file.nil?
+    # This will be used in fluent-bit.conf file to filter out logs
+    if (!@collectStdoutLogs && !@collectStderrLogs)
```
Comment (ganga1980) [inline build/common/installer/scripts/tomlparser.rb:264]: I presume, you have verified by setting both options to false and works as expected.

Comment (sarahpeiffer) [reply build/common/installer/scripts/tomlparser.rb:264]: checked and works as expected!

**Review by ganga1980 — APPROVED** [review]
lgtm

File: charts/azuremonitor-containers/values.yaml [hunk 2]
```diff
@@ -21,8 +21,8 @@ Azure:
 omsagent:
   image:
     repo: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod"
-    tag: "ciprod01312022"
-    tagWindows: "win-ciprod01312022"
+    tag: "ciprod03112022"
```
Comment (ganga1980) [inline charts/azuremonitor-containers/values.yaml:24]: @sarahpeiffer , you will have to bump chart version from 2.9.1 to 2.9.2 in chart.yaml file

**Review by ganga1980 — APPROVED** [review]

**Pull request merged**